docs/guides/render-static-scenes.mdx

```diff
@@ -97,6 +97,17 @@ curl --location --request POST \
 </TabItem>
 </Tabs>
 
+:::tip
+Scenes that are created or updated with the metadata key/value pair `"CONNECT_WORKSPACE_URN": "{YOUR_URN}"` will be added to the corresponding Connect workspace. This information can be added to the attributes section of your requests:
+```
+    "attributes": {
+      "metadata": {
+        "CONNECT_WORKSPACE_URN": "{YOUR_URN}"
+      }
+    }
+```
+:::
```

Comment:
What do you think of having a separate section for this on the [Import Data](https://developer.vertexvis.com/docs/guides/import-data-with-api) page? Seems more related to the importing of your data vs rendering. Also guessing that the import data guide will expand with details about integrating with your PDM.

Reply:
That's fair. How about this?